pkgs/by-name/un/unyaffs/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unyaffs";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "whataday";
+    repo = "unyaffs";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-FcuPaxq74gVJ6UlOhFPSMtuwUAJVV/sAxAQojhmVXCs=";
+  };
+
+  preBuild = ''
+    sed -i '54i #include <sys/sysmacros.h>' unyaffs.c
+  '';
+
+  installPhase = ''
+    install -D unyaffs -t $out/bin
+  '';
+
+  meta = with lib; {
```

Comment:
```suggestion
  meta = {
```

Reply:
In which cases should it be `meta = with lib; {` if any? since I've seen other packages done like this from which I copied this structure.
This will modify the below to:
```
    license = lib.licenses.gpl2Only;
    platforms = lib.platforms.unix;
 ```